.github/bump_chart_versions.sh

```diff
@@ -40,4 +40,5 @@ fi
 echo "update ${CHART} appVersion from ${APPVERSION} to ${LATEST}"
 bump_patch_version
 sed -i "s/^version: .*$/version: ${NEW_VERSION}/" "${CHARTFILE}"
-sed -i "s/^appVersion: .*$/appVersion: ${LATEST}/" "${CHARTFILE}"
\ No newline at end of file
+sed -i "s/^appVersion: .*$/appVersion: ${LATEST}/" "${CHARTFILE}"
+sed -i "s/${APPVERSION}/${LATEST}/" "${CHARTFILE}"
```

Comment:
probably will need an extra blank line here,

Reply:
Does shell need empty line at the end of the file?